cypress/integration/profile_spec.js

```diff
@@ -34,11 +35,11 @@ describe('The Profile Page', function () {
     cy.get('#bioInput').type(newBio)
     cy.get('input[name=name]').type(newName)
     cy.contains('Update').click()
-    cy.contains('was Updated').should('exist')
```

Comment:
Any reason to keep the commented out code?

Reply:
I forgot to go back and clean that up! I can adjust later and force push